configs/groie/mask_rcnn_r50_fpn_1x-orig.py

```diff
@@ -0,0 +1,190 @@
+# model settings
```

Comment:
This config is not necessary to be kept since it is the same as the original file.